### ISSUE TITLE
Feature req: Run multiple actions at time slot

### ISSUE THREAD

**Issue by haavares** [nielsfaber/scheduler-card#423]
For controlling my ACs, I need to set both temperature and fan speed (e.g., when going to sleep). Currently I do this by scheduling two actions a few minutes apart, but ideally two or more actions for the same slot should be supported. I believe there are other use cases for this where multiple services should be called.

- Consider allowing multiple actions to be run at a time slot (either always or as optional functionality)

**Comment by nielsfaber** [comment]
This has been discussed before.
My opinion is that combining multiple actions together will make the GUI too complex.
Also I think there are very few practical use-cases for this.

For In the past, the card supported setting `set_hvac_mode` + `set_temperature` for climate entities together.
Some users reported problems that one of the commands was ignored by their device integrations, because the commands were executed too close to eachother (the integration was still busy processing the first).
In other words, (part of) HA is not compatible with triggering multiple service calls simultaneously.

As an alternative, I suggest to create a `script` to  handle your situation, preferably with a delay between both actions.
You can also pass variables (like temperature) to a script.

**Comment by haavares** [comment]
Thanks for the quick feedback. I tried out the script solution you suggested and that worked perfectly. I was not aware that scripts accepted variables. Maybe you can add something about this for running multiple services in the documentation as well, since I would guess more users are unaware of this.

**Comment by nielsfaber** [comment]
Good idea!
Do you mind sharing your script + scheduler customization?
It seems a good example already 🙂

**Comment by haavares** [comment]
Sure. I'll refine it for a few days and share it.

**Comment by haavares** [comment]
Hi. Please find attached my script and customization. I do think I have encountered a couple of bugs in the card while experimenting with this, but the script and customization work well.

#### Customization
```yaml
customize:
  script.set_climate_livingroom:
    actions:
      - service: script.set_climate_livingroom
        name: Set climate
        icon: mdi:air-conditioner
        variables:
          hvac_mode:
            name: HVAC mode
            options:
              - value: heat
                icon: mdi:fire
              - value: cool
                icon: mdi:snowflake
              - value: 'off'
                icon: mdi:power
          temperature:
            name: Temperature
            min: 12
            max: 30
          fan_mode:
            name: Fan mode
            options:
              - value: auto
                icon: mdi:fan-auto
              - value: high
                icon: mdi:fan-speed-3
              - value: quiet
                icon: mdi:fan-speed-1
```

#### Example script to invoice multiple services for an entity:
```yaml
set_climate_livingroom:
  alias: Set climate livingroom
  description: "Sets climate parameters for scheduler-card"
  variables:
    target_entity: climate.stue_2
  sequence:
    - service: climate.set_temperature
      data:
        temperature: "{{ temperature }}"
      target:
        entity_id: "{{ target_entity }}"
    - delay:
        hours: 0
        minutes: 0
        seconds: 1
        milliseconds: 0
    - service: climate.set_fan_mode
      target:
        entity_id: "{{ target_entity }}"
      data:
        fan_mode: "{{ fan_mode }}"
    - delay:
        hours: 0
        minutes: 0
        seconds: 1
        milliseconds: 0
    - service: climate.set_hvac_mode
      target:
        entity_id: "{{ target_entity }}"
      data:
        hvac_mode: "{{ hvac_mode }}"
  mode: single
  icon: mdi:air-conditioner
```

#### Issues / scheduler-card bug?
I've been using it for two days now and it works well. It increases the WAF compared to my manual automations a lot :)

However, when using the (ios) app the variable controls do not show up. It works correctly on desktop/mobile browser, but not the app. I think it might be related to the action 'Set climate' is not selected/highlighted even on desktop when changing variables. I'll leave it open to user error as well :)

As you can see from the screenshot below, on the left side (safari) everything works, but 'Set Climate' is not selected. On the right side (app), 'Set Climate' is selected, but the variables don't show up. Let me know if you have any idea why.

<img width="1089" alt="Screen Shot 2021-06-14 at 20 45 33" src="https://user-images.githubusercontent.com/1676088/121943466-e6040400-cd51-11eb-8118-4c2c9a547179.png">

**Comment by nielsfaber** [comment]
Thanks for sharing you script configuration, I will update the docs with an example.

Regarding 
> However, when using the (ios) app the variable controls do not show up. It works correctly on desktop/mobile browser, but not the app. I think it might be related to the action 'Set climate' is not selected/highlighted even on desktop when changing variables. I'll leave it open to user error as well :)

A while ago the HA team decided to implement _lazy-loading_ for their UI component.
Which results into some elements (such as the slider bar) being only loaded to your browser when you reach some specific pages in HA.

Unfortunately it is not possible for me to interact with HA and request the loading of specific elements that scheduler-card needs.
Currently there is a workaround in the code hich should work in most cases, but I also see this failing sometimes.
I don't have any possible fix for this.

THis problem only affects the slider bar and checkboxes (as far as I know).

**Comment by nielsfaber** [comment]
FYI, I just added your example to the documentation:
https://github.com/nielsfaber/scheduler-card#triggering-multiple-actions-on-a-schedule

Thanks again!

**Comment by haavares** [comment]
Great. 2.1.4 solved the issue on iPhone btw.
